### PR TITLE
add task log & fix report url for mta

### PR DESCRIPTION
**Pull request by RichardW98**
**What this PR does / why we need it**:
add task log & fix report url for mta
Screenshot 2023-06-14 at 1 16 00 PM

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story (FLPATH-xxxx)*:
Fixes #

**Change type**
- [ ] New feature
- [ ] Bug fix
- [ ] Unit tests
- [ ] Integration tests
- [ ] CI
- [ ] Documentation
- [ ] Auto-generated SDK code

**Impacted services**
- [ ] Workflow Service
- [ ] Notification Service

**Checklist**
- [ ] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.

### CODE REVIEW

**Review by pkliczewski — COMMENTED** [review]
/lgtm

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java [hunk 1]
```diff
@@ -58,15 +58,19 @@ public WorkReport execute(WorkContext workContext) {
...
 		if (result == null) {
+			taskLogger.logErrorWithSlf4j("MTA client returned empty result with no error");
 			// unexpected
-			return new DefaultWorkReport(WorkStatus.FAILED, new WorkContext(),
+			return new DefaultWorkReport(WorkStatus.REJECTED, new WorkContext(),
 					new IllegalStateException("MTA client returned empty result with no error"));
 		}
 		else if (result instanceof Result.Failure<App> failure) {
-			return new DefaultWorkReport(WorkStatus.FAILED, workContext, failure.t());
+			taskLogger.logErrorWithSlf4j("MTA client returned Failed result");
```
Comment (masayag) [inline prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/CreateApplicationTask.java:67]: s/Failed/failed ?

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetApplicationTask.java [hunk 2]
```diff
@@ -54,15 +54,19 @@ public WorkReport execute(WorkContext workContext) {
...
 		if (result == null) {
+			taskLogger.logErrorWithSlf4j("MTA client returned empty result with no error");
 			// unexpected
-			return new DefaultWorkReport(WorkStatus.FAILED, new WorkContext(),
+			return new DefaultWorkReport(WorkStatus.REJECTED, new WorkContext(),
 					new IllegalStateException("MTA client returned empty result with no error"));
 		}
 		else if (result instanceof Result.Failure<App> failure) {
-			return new DefaultWorkReport(WorkStatus.FAILED, workContext, failure.t());
+			taskLogger.logErrorWithSlf4j("MTA client returned Failed result");
```
Comment (masayag) [inline prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetApplicationTask.java:63]: s/Failed/failed

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/SubmitAnalysisTask.java [hunk 3]
```diff
@@ -59,15 +59,19 @@ public WorkReport execute(WorkContext workContext) {
...
 		if (result == null) {
+			taskLogger.logErrorWithSlf4j("MTA client returned empty result with no error");
 			// unexpected
-			return new DefaultWorkReport(WorkStatus.FAILED, new WorkContext(),
+			return new DefaultWorkReport(WorkStatus.REJECTED, new WorkContext(),
 					new IllegalStateException("MTA client returned result with no error"));
 		}
 		else if (result instanceof Result.Failure<TaskGroup> failure) {
-			return new DefaultWorkReport(WorkStatus.FAILED, workContext, failure.t());
+			taskLogger.logErrorWithSlf4j("MTA client returned Failed result");
```
Comment (masayag) [inline prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/SubmitAnalysisTask.java:68]: s/Failed/failed

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java [hunk 4]
```diff
@@ -85,25 +85,29 @@ public WorkReport execute(WorkContext workContext) {
...
 		else if (result instanceof Result.Failure<TaskGroup> failure) {
-			return new DefaultWorkReport(WorkStatus.FAILED, workContext, failure.t());
+			taskLogger.logErrorWithSlf4j("MTA client returned failed result");
+			return new DefaultWorkReport(WorkStatus.REJECTED, workContext, failure.t());
 		}
 		else if (result instanceof Result.Success<TaskGroup> success) {
 			if ("Ready".equals(success.value().state()) && success.value().tasks() != null
 					&& "Succeeded".equals(success.value().tasks()[0].state())) {
-				String reportURL = "%s/hub/applications/%d/bucket/%s".formatted(serverUrl,
+				String reportURL = "%s/hub/applications/%d/bucket%s".formatted(serverUrl,
```
Comment (masayag) [inline prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java:100]: not for today, but perhaps this one can use something like URLBuilder to make sure the URL is valid.

**Review by masayag — APPROVED** [review]

File: prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java (hunk 4 above)
Comment (RichardW98) [reply prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/GetAnalysisTask.java:100]: yea agree

**Pull request merged**